web/admin/src/Admin.js

```diff
@@ -201,6 +221,36 @@ export default class Admin extends PureComponent {
     )
   }
 
+  getAdmins = keys => {
+    const attendeeClickPromises = keys.map(result =>
+      client
+        .getAttendee(result)
+        .then(attendee => ({ ...attendee }))
+        .catch(err => result),
+    )
+    Promise.all(attendeeClickPromises).then(newResults => {
+      this.setState({ admins: newResults })
+    })
+  }
+
+  onAdminSelected = attendee => {
+    const tokenRef = this.props.fbc.database.private
+      .adminableUsersRef(attendee.id)
+      .child('adminToken')
+    this.setState()
+    this.props.fbc.getLongLivedAdminToken().then(token => tokenRef.set(token))
+    this.props.fbc.database.private
+      .adminRef('adminUrl')
+      .set(window.location.href + this.adminScreenUrlSave())
```

Comment:
What happens if an admin URL is saved, then the extension is updated to a later version?  We would be relying on a URL of a specific past version in that case.

Reply:
Theoretically yes. Not sure the best work around for this... This was originally why I did not hardcode the url into the app and just save the long live token